utils/generate_types/overrides-testReporter.d.ts

```diff
@@ -14,9 +14,15 @@
  * limitations under the License.
  */
 
-import type { FullConfig, FullProject, TestStatus, Metadata } from './test';
+import type { FullConfig, FullProject, TestStatus, Metadata, PlaywrightTestOptions, PlaywrightWorkerOptions } from './test';
 export type { FullConfig, TestStatus, FullProject } from './test';
```

Comment:
Do we want to make `FullConfig` an alias to `ReporterConfig` instead?

Reply:
As discussed in the meeting, let's preserve FullConfig for reporters and have ConfigInWorker for test/worker info (breaking change).